### PR TITLE
addition of two ploynomial #67

### PR DESCRIPTION
**Pull request by ashishugi**
Add two polynomial using linked list .

### CODE REVIEW

**Review by rishabhgarg25699 — CHANGES_REQUESTED** [review]
@ashishugi Do proper indentation.

**Comment by ashishugi** [comment]
hey could you be more specific , which indentation you are taking about

File: addition.cpp [hunk 1]
```diff
@@ -0,0 +1,112 @@
+#include<bits/stdc++.h>
+using namespace std;
+struct node
+{
+ struct node *next;
```
Comment (rishabhgarg25699) [inline addition.cpp:5]: 4 space inden.

File: addition.cpp [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+{
+  struct node *nw,*ptr;
+  nw=(struct node *)malloc(sizeof(struct node));
+  nw->next=NULL;
+  nw->coef=coef;
+  nw->expo=expo;
+
+  if(first==NULL)
+  {
+
```
Comment (rishabhgarg25699) [inline addition.cpp:19]: Extra space

File: addition.cpp [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+  {
+    if(p->expo==q->expo)
+    {
+       first=insert(first,(p->coef+q->coef),p->expo);
+       p=p->next;
+       q=q->next;
+    }
+    else if(p->expo>q->expo)
+    {
+     first=insert(first,p->coef,p->expo);
```
Comment (rishabhgarg25699) [inline addition.cpp:42]: No indentation

**Comment by rishabhgarg25699** [comment]
Many indentations problem

**Comment by ashishugi** [comment]
okay i will be doing it .

**Comment by ashishugi** [comment]
done this , please check out.

File: addition.cpp [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+   cin>>expo;
+   firstp=insert(firstp,coef,expo);
+   cout<<"Do you want to enter more Y/N:";
+   cin>>ch;
+  }
+  ch='y';
+  cout<<"\n Now for the 2nd equation";
+  while(ch=='y')
+  {
+   cout<<"enter the coefficient : "<<endl;
```
Comment (rishabhgarg25699) [inline addition.cpp:95]: Where is 4 space ?

File: addition.cpp [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+   cin>>ch;
+ }
+   firstr=add(firstp,firstq,firstr);
+   cout<<"the first euation is: "<<endl;
+   display(firstp);
+   cout<<" the 2nd equation is:\t"<<endl;
+   display(firstq);
+   cout<<"the resultant equation is : \t"<<endl;
+   display(firstr);
+
```
Comment (rishabhgarg25699) [inline addition.cpp:110]: No extra space

File: addition.cpp [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+    
+}
```
Comment (rishabhgarg25699) [inline addition.cpp:112]: Many mistakes are there.

**Pull request closed without merging**